releasenotes/notes/generate-experiments-2ac773442132c78d.yaml

```
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Added a module, :mod:`circuit_knitting.cutting.cutting_experiments`, which is intended to hold
+    functions used for generating the quantum experiments needed for circuit cutting. This module
+    will initially hold one function, :func:`circuit_knitting.cutting.cutting_experiments.generate_cutting_experiments`,
```

```suggestion
    will initially hold one function, :func:`~circuit_knitting.cutting.cutting_experiments.generate_cutting_experiments`,
```
(probably don't need to display the whole name here)